docs/arch/idea-front_end_ui_scaffolding.md

```diff
@@ -28,16 +28,19 @@ You're also absolutely right that all the Sprint 5 UI tasks can be implemented w
 ## 🧱 Project Layout Example
 
 ```bash
-clarifai-ui/
-├── gradio/
-│   ├── main.py                # Launches Gradio app with tabs
-│   ├── config_panel.py        # Model selection, thresholds, windows
-│   ├── scheduler_panel.py     # Enable/disable jobs
-│   ├── automation_control.py  # Pause/resume
-│   └── components/            # Reusable Gradio UI elements (optional)
+clarifai-monorepo/
+├── settings/
+│   ├── ./settings/clarifai.config.yaml   # Central configuration file
+│   └── prompts/               # User-editable prompts
```

Comment:
I believe this is incorrect. In the proposed solution in the issue, there was:

+├── settings/
+│   ├── clarifai.config.yaml   # Central configuration file

in /docs/arch/idea-front_end_ui_scaffolding.md, and I see no reason for it to be changed. I do notice elsewhere in your pr you use "./settings/clarifai.config.yaml" where the proposed change was "settings/clarifai.config.yaml". Please justify this in an explanation before I continue my review.